app/views/users/show.html.erb

```diff
@@ -4,6 +4,9 @@
     <h3 class='user-bio-title'>Bio</h3>
     <p class='user-bio-text'><%= @user.bio %></p>
   </div>
+  <p>
+    <%= link_to "Creat New post", new_user_post_path(@user) %>
+  </p>
```

Comment:
- Please note that after populating the database with data, the page is still rendering an empty page. Kindly fix this 👍🏽
![Screenshot from 2023-03-04 17-37-35](https://user-images.githubusercontent.com/44624138/222917994-5d1c1946-9568-4068-bd30-5f5019a1a6f4.png)

Reply:
![screencapture-127-0-0-1-3000-2023-03-04-21_47_53](https://user-images.githubusercontent.com/109666020/222918464-49944f00-77bf-4dbb-be60-d762652825d4.png)


- It's working on my side please add the data using `rails console`

Reply:
@rotimiazeez first add the users by `rails console` then you will see the data on website.
browser not allowed you to add user, you can just add post from browser

Reply:
Hi @Mujeeb4582,
Thanks for reaching out, I try adding data via rails console and the seed file in your dB folder but it didn't render them on the app

Reply:
@Mujeeb4582,
Kindly submit again since you cleared the post_counter validation and all is working as expected, your PR will be approved 👍🏼